src/__test__/components/AddAssetComponent.test.js

```diff
@@ -189,6 +189,32 @@ describe('<AddAssetContainer />', () => {
     expect(wrapper.state().specs.memory).toEqual('16');
   });
 
+  it('should submit the form when form is filled', () => {
+    specsComponentSetup();
+    const returnSpecsState = () => wrapper.state().specs;
+    wrapper.setProps({
+      createAsset: () => returnSpecsState()
+    });
+
+    expect(returnSpecsState()).toEqual(wrapper.state().specs);
+
+    wrapper.find('DropdownComponent .item').at(3).simulate('click');
+    wrapper.find('input[type="radio"][name="processorType"]').at(0).simulate('change');
```

Comment:
You can add a data attribute to the inputs named `processorType` and use that for testing. This way, you don't have to specifically get the radio input of the `processorType`. E.g.
In `My component`, I may have the following:
```
<label data-test="processor-type-input">
   <input type="radio" name="my-radio-button" checked />
   Testing 1
</label>
<label data-test="processor-type-input">
   <input type="radio" name="my-radio-button" />
   Testing 2
</label>
```

Then in `My test`, I'll do the following:
```
wrapper.find('[data-test="processor-type-input"]').first().simulate('change')
```